pages/embedded/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { GetServerSideProps } from 'next';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import { useTranslation } from 'next-i18next';
+import Layout from '../../components/Layout/Layout';
+import PageHead from '../../components/PageHead/PageHead';
+import HeroBlockTechnology from '../../components/HeroBlockTechnology/HeroBlockTechnology';
+import Points from '../../components/Points/Points';
+import Tasks from '../../components/Tasks/Tasks';
+import Cases from '../../components/Cases/Cases';
+import Cta from '../../components/Cta/Cta';
+import Stack from '../../components/Stack/Stack';
+import Payment from '../../components/Payment/Payment';
+import Cooperation from '../../components/Cooperation/Cooperation';
+import ServicesTechnology from '../../components/ServicesTechnology/ServicesTechnology';
+import FormBlock from '../../components/FormBlock/FormBlock';
+
+export default function EmbeddedPage() {
+  const { t } = useTranslation('common');
+
+  return (
+    <>
+      <PageHead
+        seoData={{
+          seo: {
+            title: t('title'),
+            description: t('description'),
+          },
+          keywords: t('keywords'),
+          metaTags: [],
+          structuredData: '',
+          additionalCode: '',
+        }}
+      />
+      <Layout mainClassName="embedded">
```

Comment:
I think it is worth to change BEM classes for other pages as well, for example 
<img width="564" alt="Screenshot 2024-06-05 at 10 10 10 AM" src="https://github.com/fpandyz/TourmalineCore.Home/assets/39587602/6b26b26d-f599-451e-aa13-3bac487d61b5">
<img width="534" alt="Screenshot 2024-06-05 at 10 10 30 AM" src="https://github.com/fpandyz/TourmalineCore.Home/assets/39587602/80076f62-5508-4b22-b960-08e3259acab3">